repos.yaml

```diff
@@ -167,6 +167,13 @@
   channel: 'net-certmanager'
   assignees: knative-sandbox/networking-wg-leads
 
+# knative-sandbox/kn-plugin-*, sorted alphabetically
+- name: 'knative-sandbox/kn-plugin-sample'
+  meta-organization: 'knative-sandbox'
+  fork: 'knative-automation/kn-plugin-sample'
+  channel: 'client'
+  assignees: 'knative/client-writers'
```

Comment:
I believe the org must match for this to work.

Reply:
I have seen samples where this is not the case, so I hope this works as is:

```
# knative-sandbox (reconciler test)
- name: 'knative-sandbox/reconciler-test'
  meta-organization: 'knative-sandbox'
  fork: 'knative-automation/reconciler-test'
  channel: 'test'
  assignees: knative/eventing-writers knative/serving-writers
```

Reply:
I am fairly sure that it only works for reviewers (`/cc`), and only then if the reviewer group has write access.

95% chance it's just broken for reconciler-test.

Reply:
https://github.com/knative-sandbox/reconciler-test/pull/90#issuecomment-779528129

Reply:
Thanks for the headsup !  I can change this to `knative-sandbox/client-writers` but I don't know whether this is defined. In the peribolos config I only see a reference to "Client Writers", not sure if this translate to a GitHub team 'client-writers'. Well, lets try it nevertheless ...